adapter/BUILD

```diff
@@ -27,3 +28,17 @@ go_library(
         "//pkg/adapter:go_default_library",
     ],
 )
+
+inventory_library(
```

Comment:
In fact, if the user needs to edit the BUILD file anyway, then I don't see the point in generating inventory.go, they can just edit that themselves too.

The general idea I was going for here is that an operator could take an unchanged Mixer repo and augment it with their suite of adapters. If we're having the operator change files in the original Mixer repo, we've somewhat eroded that value prop.